gcloud/_helpers.py

```diff
@@ -394,6 +394,30 @@ def _to_bytes(value, encoding='ascii'):
         raise TypeError('%r could not be converted to bytes' % (value,))
 
 
+def _from_bytes(value):
+    """Converts bytes to a string value, if necessary.
+
+    Args:
+        value: The string/bytes value to be converted.
+
+    Returns:
+        The original value converted to unicode (if bytes) or as passed in
+        if it started out as unicode.
+
+    Raises:
+        ValueError if the value could not be converted to unicode.
+
+    :type value: bytes
+    :param value: bytes value to attempt string conversion on.
+    """
```

Comment:
This is not a Sphinx-style docstring (it is Google-style). Remove the `Args`, `Returns` and `Raises` sections / convert them into relevant sections in Sphinx-style.